pkg/test/client.go

```diff
@@ -32,9 +32,9 @@ var _ FrameworkClient = &frameworkClient{}
 
 type FrameworkClient interface {
 	Get(gCtx goctx.Context, key dynclient.ObjectKey, obj runtime.Object) error
-	List(gCtx goctx.Context, list runtime.Object, opts ...dynclient.ListOptionFunc) error
+	List(gCtx goctx.Context, list runtime.Object, opts ...dynclient.ListOption) error
 	Create(gCtx goctx.Context, obj runtime.Object, cleanupOptions *CleanupOptions) error
-	Delete(gCtx goctx.Context, obj runtime.Object, opts ...dynclient.DeleteOptionFunc) error
+	Delete(gCtx goctx.Context, obj runtime.Object, opts ...dynclient.DeleteOption) error
```

Comment:
Is this mentioned in the `Breaking Changes` section of the CHANGELOG?
Since this a public API as part of the test framework we should mention it.

Not sure if we have any actual documentation to update for the FrameworkClient @AlexNPavel

Reply:
Done in CHANGELOG.md